README.md

```diff
@@ -31,9 +31,10 @@ you can install the type stubs using
     $ pip install types-six types-requests
 
 These PyPI packages follow [PEP 561](http://www.python.org/dev/peps/pep-0561/)
-and are automatically generated by typeshed internal machinery. Also starting
-from version 0.900 mypy will provide an option to automatically install missing
-type stub packages (if found on PyPI).
+and are automatically released (daily) by
```

Comment:
If I understood correctly, the release always happens within 3 hours, but it would be good if another typeshed maintainer confirms this.

Reply:
I'm pretty sure 3 hours is right. Elsewhere you linked to the crontab and the cron runs every 3 hours.

Reply:
Oh you right. I read the cron too fast and saw every day at 3, where it's "every hours divisible by 3" so 8 times a day

Reply:
You can check how often the CI runs: https://github.com/typeshed-internal/stub_uploader/actions/workflows/update_stubs.yml

I was put off by the name of the workflow: "Daily update of stubs from typeshed to PyPI".

Reply:
Ok, fixed :)